toucan_connectors/trello/trello_connector.py

```diff
@@ -44,10 +44,23 @@ class Fields(str, Enum):
     labels = 'labels'
 
 
+class Filter(str, Enum):
+    """
+    cf https://developer.atlassian.com/cloud/trello/rest/#api-boards-id-cards-filter-get
+    """
```

Comment:
IMO, this is a very generic class name. May we rename it `CardsFilter`, `CardStatus`, or something like it?

```suggestion
class CardsFilter(str, Enum):
    """
    A filter to match cards in a specific state on a board.
    
    https://developer.atlassian.com/cloud/trello/rest/#api-boards-id-cards-filter-get
    """
```